packages/node/src/lib/process-request.ts

```diff
@@ -112,7 +113,7 @@ function parseRequestBody(body: string, mimeType: string): Record<string, unknow
     try {
       return JSON.parse(body);
     } catch (err) {
-      // no-op
+      logger.error({ message: 'Error parsing request body JSON (process-request)', err });
```

Comment:
@AndriiAndreiev if you're going to (partially) reference a file here, I'd do it throughout. Realistically, there's not a whole lot folks can do here, so I'd probably just omit the file reference.